linalg/sparsemat.hpp

```diff
@@ -631,19 +645,19 @@ class SparseMatrix : public AbstractSparseMatrix
    virtual ~SparseMatrix()
    {
       Destroy();
-#ifdef MFEM_USE_CUDA
-      if (useCuSparse)
+#ifdef CUDA_or_HIP
+      if (useGPUSparse)
```

Comment:
Move the destructor definition to the `.cpp` file.

Reply:
done